### DISCUSSION TITLE
Change last button text to another word?

### DISCUSSION THREAD

**Discussion by caotrongtin99** [gilbarbara/react-joyride#856] [Q&A] [answered]
HI @gilbarbara ,

Can I change the last button text from 'Last' to another word?

**Comment by gilbarbara** [comment, marked as answer]
Hey @caotrongtin99 

Have you checked the [documentation](https://docs.react-joyride.com/props)?
You can use the `locale` prop.

**Comment by ajaykumar1071999** [comment]
you can try this

locale={{
          skip: "Skip",
          last:"Done",

        }}
